http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala

```diff
@@ -726,6 +728,7 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
           logger.error(s"Wsm deleteVm job failed due to ${resp.getErrorReport.getMessage}") >> F.raiseError[Unit](
             AzureRuntimeDeletionError(
               params.runtime.id,
+              None,
```

Comment:
For VM deletion polling, the diskID is not part of the parameters (same for storage container deletion below). If this is just for logging then it is okay, but if raising this error triggers database status changes then I will need to change that.

Reply:
hmm yeah I think this is then picked up by the handleAzureRuntimeDeletionError func in the LeoPubsubMessageSubscriber, so we might need these to not be `None`...

Reply:
done